interfaces/functions.yaml

```diff
@@ -1601,7 +1601,8 @@ igraph_community_leiden:
     PARAMS: |-
         GRAPH graph, OPTIONAL EDGEWEIGHTS weights,
         OPTIONAL VERTEXWEIGHTS vertex_weights,
-        REAL resolution, REAL beta, BOOLEAN start, OPTIONAL INOUT VECTOR_INT membership,
+        REAL resolution, REAL beta, BOOLEAN start, INTEGER n_iterations, 
```

Comment:
Should `n_interations` have a default value here?

For `beta`, the docs recommend 0.01. Should it have a default value here?

Reply:
Yes, good point. The recommended default is actually 2 iterations. Not sure if I applied that recommendation currently in the higher level interfaces.